symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/main/java/com/symphony/bdk/bot/sdk/symphony/DatafeedClientImpl.java

```diff
@@ -10,6 +10,7 @@
 import listeners.ConnectionListener;
 
 @Service
+@ConditionalOnProperty(value = "datafeed.polling.disabled", havingValue="false", matchIfMissing=true)
```

Comment:
@marcus-secato-symphony import is missing right?

Reply:
yes, fixing that.